cmd/list.go

```diff
@@ -285,7 +287,7 @@ func displayListPayload(
 		df = df.CBind(dataframe.New(colSeries))
 	}
 
-	df, err := formatter.SortDataFrame(df, sortColumn, true)
+	df, err := formatter.SortDataFrame(df, sortColumn, true, readableToRaw, humanReadable)
```

Comment:
This approach works nicely! I guess the only problem is that `SortDataFrame()` was very generic before and now it's a bit coupled with `list` command. Since it's the only place where it's used it might be ok. Also, if we would use this function in some other command probably this logic would be beneficial to have. 
Just to document that another approach would be to create separate series for `sortable_run_number` and `sortable_size` before sorting and to drop it after